test_requirements.txt

```diff
@@ -10,3 +10,9 @@ requests-mock==1.11.0
 sphinx==7.1.2
 sphinx-rtd-theme==2.0.0
 sphinxcontrib-apidoc==0.4.0
+
+# faker is a dependency of factory-boy, but factory-boy sets a very low floor
```

Comment:
I was getting "Unknown formatter 'enum' with locale 'en_US'" errors. This was fixed in 15.1.0, but factory-boy only requires version >=0.7.0